app/views/layouts/godmin/_layout.html.erb

```diff
@@ -7,8 +7,9 @@
     <%= csrf_meta_tags %>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <%= yield :head %>
+    <img src="/logo.jpg" alt="logo" width="128" height="128"/>
```

Comment:
I prefer to set dimensions with a `style` attribute. Also, this image is not a square, so I would try setting this to:
```
<img src="/logo.jpg" alt="logo" style="height: 128px; width: auto;"/>
```
but hey, this is Rails, we can use a helper:
```
<%= image_tag 'logo.jpg', style: 'height: 128px; width: auto;'%>
```

The logo itself should be saved in `app/assets/images/`